src/dfx/src/lib/network/network_opt.rs

```diff
@@ -1,24 +1,33 @@
-use clap::Args;
+use clap::{ArgGroup, Args};
 
 #[derive(Args, Clone, Debug, Default)]
+#[clap(
+group(ArgGroup::new("network-select").multiple(false)),
+)]
 pub struct NetworkOpt {
     /// Override the compute network to connect to. By default, the local network is used.
     /// A valid URL (starting with `http:` or `https:`) can be used here, and a special
     /// ephemeral network will be created specifically for this request. E.g.
     /// "http://localhost:12345/" is a valid network name.
-    #[arg(long, global = true, conflicts_with("playground"))]
+    #[arg(long, global = true, group = "network-select")]
```

Comment:
```suggestion
    #[arg(long, global(true), group = "network-select")]
```

Consistent with following options.